py3status/modules/check_tcp.py

```diff
@@ -29,23 +31,23 @@ class Py3status:
     format = '{host}:{port} {state}'
     host = 'localhost'
     port = 22
+    string_down = 'DOWN'
+    string_up = 'UP'
 
-    def check_tcp(self):
-        response = {'cached_until': self.py3.time_in(self.cache_timeout)}
+    def post_config_hook(self):
+        self.color_on = self.py3.COLOR_UP or self.py3.COLOR_GOOD
+        self.color_off = self.py3.COLOR_DOWN or self.py3.COLOR_BAD
```

Comment:
to me `color_on/off` make more sense - it is sort of messy as we override these but sort of not as they are not really used.